tools/run_tests/xds_k8s_test_driver/README.md

```diff
@@ -125,22 +124,199 @@ python3 -m tests.baseline_test \
   --server_xds_port="$(($RANDOM%1000 + 34567))"
 ```
 
+## Local development
```

Comment:
Can we also mention workload identity here? Or point to the official doc. I found workload identity is essential to all test runs.

Without workload identity enabled:

```
WARNING: 2021/05/24 07:42:15 [xds] [xds-client 0xc0003f6400] ADS stream is closed with error: xds: stream.Recv() failed: rpc error: code = PermissionDenied desc = Request had insufficient authentication scopes.
```

Workload identity not properly assigned.

```
WARNING: 2021/05/24 07:18:30 [xds] [xds-client 0xc0002c0400] xds: ADS stream creation failed: rpc error: code = Unauthenticated desc = transport: compute: Received 403 `Unable to generate access token; IAM returned 403 Forbidden: The caller does not have permission
This error could be caused by a missing IAM policy binding on the target IAM service account.
For more information, refer to the Workload Identity documentation:
	https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#authenticating_to
```

Reply:
Good point. I'll add some notes, but I don't want to go too much into details, because my next PRs will make this irrelevant.